draft-ietf-quic-http.md

```diff
@@ -317,8 +318,14 @@ client abort transmission of a request without error by triggering a QUIC
 STOP_SENDING with error code HTTP_EARLY_RESPONSE, sending a complete response,
 and cleanly closing its streams. Clients MUST NOT discard complete responses as
 a result of having their request terminated abruptly, though clients can always
-discard responses at their discretion for other reasons.  Servers MUST NOT
-abort a response in progress as a result of receiving a solicited RST_STREAM.
+discard responses at their discretion for other reasons.
+
+Changes to the state of a request stream, including receiving a RST_STREAM
+(regardless of error code), do not affect the state of the server's response.
+Servers MUST NOT abort a response in progress solely due to a state change on
+the request stream. However, if the request stream terminates without containing
+a fully-formed HTTP request, the server SHOULD abort its response with the error
```

Comment:
s/fully-formed/usable/ perhaps.

A server can often produce a response based purely on header fields.  That's the case that we specifically want to support with STOP_SENDING (a 404 on a POST).  This implies that the message needs to be complete, which isn't necessary.